src/Fields/Fields.jl

```diff
@@ -393,5 +393,64 @@ function set!(f::Function, field::Field, args = ())
     return nothing
 end
 
+#=
+This function can be used to truncate the printing
+of ClimaCore `Field` types, which can get rather
+long.
+
+# Example
+```
+import ClimaCore
+ClimaCore.Fields.truncate_printing_field_types() = true
+```
+=#
+truncate_printing_field_types() = false
+
+function Base.show(io::IO, ::Type{T}) where {T <: Fields.Field}
+    if truncate_printing_field_types()
+        print(io, print_truncated_field_type(T))
+    else
+        invoke(show, io, print_truncated_field_type(T))
+    end
+end
+
+# Defined for testing
+function print_truncated_field_type(::Type{T}) where {T <: Fields.Field}
```

Comment:
don't call this `print_truncated_field_type` if it doesn't print anything

Reply:
Changed to `truncated_field_type_string`.